modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -44,20 +46,30 @@ The procedures and commands for vector indexes are listed in the following table
 |===
 | Usage | Procedure/Command | Description
 
-| Create vector index.
+| Create node vector index.
 | `+CREATE VECTOR INDEX ...+`
-| label:new[Introduced in 5.15] Create a vector index for the specified label and property with the given vector dimensionality using the given similarity function.
+| label:new[Introduced in 5.15] Create a node vector index for the specified label and property with the given vector dimensionality using the given similarity function.
 See the xref:indexes/search-performance-indexes/managing-indexes.adoc#create-indexes[`CREATE INDEX`] command for more details. Replaces {link-procedures-reference}#procedure_db_index_vector_createNodeIndex[`db.index.vector.createNodeIndex`].
 
+| Create relationship vector index.
+| `+CREATE VECTOR INDEX ...+`
+| label:new[Introduced in 5.18] Create a relationship vector index for the specified relationship type and property with the given vector dimensionality using the given similarity function.
+See the xref:indexes/search-performance-indexes/managing-indexes.adoc#create-indexes[`CREATE INDEX`] command for more details.
```

Comment:
Labels in a table should go at the end - I have an open PR that fixes this for the rest of the manual.
```suggestion
| Create a node vector index for the specified label and property with the given vector dimensionality using the given similarity function.
See the xref:indexes/search-performance-indexes/managing-indexes.adoc#create-indexes[`CREATE INDEX`] command for more details. Replaces {link-procedures-reference}#procedure_db_index_vector_createNodeIndex[`db.index.vector.createNodeIndex`].  label:new[Introduced in 5.15]

| Create relationship vector index.
| `+CREATE VECTOR INDEX ...+`
| Create a relationship vector index for the specified relationship type and property with the given vector dimensionality using the given similarity function.
See the xref:indexes/search-performance-indexes/managing-indexes.adoc#create-indexes[`CREATE INDEX`] command for more details. label:new[Introduced in 5.18]
```